editor.planx.uk/src/pages/FlowEditor/components/Team/components/AddNewEditorModal.tsx

```diff
@@ -24,13 +24,15 @@ import { optimisticallyUpdateMembersTable } from "./lib/optimisticallyUpdateMemb
 export const AddNewEditorModal = ({
   showModal,
   setShowModal,
-  setShowToast,
+  setShowSuccessToast,
+  setShowErrorToast,
```

Comment:
This is the sort of thing I hope to get rid of with a `useToast` hook in another PR 😁